ee/tuf/autoupdate.go

```diff
@@ -314,7 +399,7 @@ func (ta *TufAutoupdater) checkForUpdate() error {
 	// Check for and download any new releases that are available
 	updatesDownloaded := make(map[autoupdatableBinary]string)
 	updateErrors := make([]error, 0)
-	for _, binary := range binaries {
+	for _, binary := range binariesToCheck {
```

Comment:
I think we want to be cautious here. Because some of these trigger restarts, we should think about what order we want to do these in, and if we need to separate them temporally.

Reply:
This is already addressed -- the order here does not matter. After we've finished downloading the updates, we check first to see if launcher was updated, and do a full reload if so (since that will also reload osqueryd). Otherwise, we check to see if osqueryd was updated, and restart it if so.